README.md

```diff
@@ -11,7 +11,3 @@ $ git clone https://github.com/bachand/battlestation.git
 $ cd battlestation
 $ ./bin/battlestation setup
 ```
-
-## Gotchas
-
-- [`setup`](bin/setup) must be run from a directory underneath this repo.
```

Comment:
The supported way to invoke bin/setup is via through the `setup` command shown earlier. The `setup` command ensures that the legacy setup script is invoked from within this repo.